<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myReceive</class>
 <widget class="QWidget" name="myReceive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我收的</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
                QLabel {
                    color: #333;
                    font: 14px &quot;幼圆&quot;;
                    font-weight: bold;
                }

                QLineEdit,
                QComboBox  {
                    padding: 8px;
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QTableWidget {
                    font: 14px &quot;幼圆&quot;;
                    background-color: #cdefff;
                }

                QTableWidget::item {
                    font: 14px &quot;幼圆&quot;;
                }

                QHeaderView {
                    font: bold 16px &quot;幼圆&quot;;
                }
                
                QTableWidget::item:selected {
                    background-color: #a6a6a6;
                }

                QTableWidget QHeaderView::section {
                    background-color: #65a3ff;
                    color: white;
                    font-weight: bold;
                    font-size: 16px;
                    padding: 4px;
                }

                QTableWidget::item {
                    padding: 10px;
                }

                QPushButton {
                    padding: 8px 16px;
                    font-size: 15px;
                    font-weight: bold;
                    border: 1px solid #4CAF50;
                    border-radius: 4px;
                    color: #fff;
                    background-color: #4CAF50;
                }

                 QPushButton:hover {
                    background-color: #45a049;
                    border: 1px solid #45a049;
                }

                QTextEdit {
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QScrollBar:vertical {
                    border: 1px solid #ccc;
                    background: #f0f0f0;
                    width: 10px;
                    margin: 0px 0px 0px 0px;
                }

                QScrollBar::handle:vertical {
                    background: #4CAF50;
                    min-height: 20px;
                }
            </string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>快递信息</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>521</width>
      <height>401</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>46</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>28</number>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>快递单号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前位置</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>预计送达时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>完整路线</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>111</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>返回上一级</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>用户操作</string>
   </property>
   <widget class="QPushButton" name="refuseBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>111</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>拒绝签收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>111</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>确认签收</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
